_posts/2020-01-12-hhvm-4.92.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: "HHVM 4.92"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.92 is released! This release marks the end of support for 4.83;
+HHVM 4.84&ndash;4.91 remain supported, as do the 4.56 and 4.80 LTS releases.
+
+(Note: HHVM 4.85, 4.89 and 4.90 were skipped.)
+
+# Highlights
+
+- If a class A is a supertype of class/interface B via `require extends`, it
+  is now able to call its' own private methods on instances of B.
+- Corrected error line number for reified generic errors.
```

Comment:
Is this a breaking change (FIXMEs may need to be moved) or is this just a change in some secondary line number?